### PR TITLE
Allow async loading

### PR DESCRIPTION
**Pull request by kesla**
R = @iefserge
T = minor (?)

This adds the possibility to load the script async - way after the platform.js script has been loaded. I also believe this is good for performance. Yey

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -16,9 +16,17 @@ export function render ({ props }) {
...
-    <div class='g-ytsubscribe' data-channelid={props['channel-id'] || 'Youtube'} data-layout='default' data-count='hidden' data-onytevent={fn}></div>
+    <div id={id}>
+      <div class='g-ytsubscribe' data-channelid={props['channel-id'] || 'Youtube'} data-layout='default' data-count='hidden' data-onytevent={fn}></div>
+    </div>
   );
 }
 
-export default { render };
+export function afterMount (component, el) {
+  window.gapi.ytsubscribe.go(el.id);
```
Comment (iefserge) [inline index.js]: let's check if api is loaded before using

**Comment by iefserge** [comment]
Nice!

File: index.js (hunk 1 above)
Comment (kesla) [inline index.js]: what should we do if it's not loaded?

Comment (kesla) [inline index.js]: I guess I can do something like

```js
if (!window.gapi) {
  console.error('deku-youtube-subscribe-button: Please add <script src="https://apis.google.com/js/platform.js"></script> to your html (before deku-youtube-subscribe-button is run'D);
}

Comment (iefserge) [inline index.js]: yeah that's awesome!

**Comment by kesla** [comment]
@iefserge Looks good? now? You'd like to merge & release this? :dancers:

**Comment by iefserge** [comment]
@kesla do you think it needs a window object check? afterMount never runs on server, right?

**Comment by kesla** [comment]
@iefserge correct

**Pull request merged**